.github/workflows/deployment-actions.yml

```diff
@@ -0,0 +1,140 @@
+name: Draft Deployment
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  deploy_web:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Deploy on Web
+        env:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          npm install
+          npm run deploy:web -- --repo=https://${{secrets.GH_TOKEN}}@github.com/CATcher-org/CATcher.git
+
+  draft_release:
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.set_upload_url.outputs.upload_url }}
```

Comment:
Quick question! How would the workflow know which version num to set and which upload url to send to? Is it part of the secrets as well? ><

Reply:
The version number is based off the package.json's version number (line 35). After creating the draft_release (in line 42), we can retrieve the URL as seen in line 51.

Reply:
Alright! Took a while to look at the documentation for both actions! 

Thanks!